index.html

```diff
@@ -0,0 +1,158 @@
+              <span class="timestamp">2h ago</span>
+              <div class="text">petition to have <span class="tag">Crowley</span> talk in his "Did you miss me??? I bet you did!" voice all through season 3 <span class="tag">GoodOmens</span> <span class="mention">neilhimself</span> <span class="mention">DavidTennant</span> <span class="tag">veryRogerIAmInLoveWithMyCarTaylorOfHim</span></div>
+              <div class="embeddedMedia">
+                <img src="https://64.media.tumblr.com/c64dbb5eeaba1a0243a6e21a999cb844/6ae10afce9054ab6-5c/s540x810/ab5726dee539ecffd6fbf24a2e81bc89eff1cd54.gifv" alt="">
+                <img src="https://64.media.tumblr.com/b4399b25efb5686b929ae698f519cc0d/6ae10afce9054ab6-60/s540x810/24bc426386bb25dd3f5d58e572d8e3ce225cb704.gifv" alt="">
+              </div>
+              <div class="postOptions">
+                <div><img src="assets/countReplies.svg" alt=""><span>3</span></div>
+                <div><img src="assets/countRetweet.svg" alt=""><span>10</span></div>
+                <div><img src="assets/countLikes.svg" alt=""><span>100</span></div>
+                <div><img src="assets/countViews.svg" alt=""><span>213</span></div>
+                <div><img src="assets/countShares.svg" alt=""></div>
+              </div>
+            </div>
+          </div>
+
+          <div class="tweet">
+            <div class="profileImg"><img src="https://64.media.tumblr.com/7d03ff145030be1e371bd5a86489f72b/f137824e42cd9969-3c/s128x128u_c1/6eadb5eec2051bf8d501eeec0c0f8b0f1f3e5989.jpg" alt=""></div>
+            <div class="content">
+              <span class="profileName">Gigi</span>
+              <span class="handle">glitterypin</span>
+              <span class="timestamp">3h ago</span>
+              <div class="text">dear lord I need a support group</div>
+              <div class="postOptions">
+                <div><img src="assets/countReplies.svg" alt=""><span>3</span></div>
+                <div><img src="assets/countRetweet.svg" alt=""><span>10</span></div>
+                <div><img src="assets/countLikes.svg" alt=""><span>100</span></div>
+                <div><img src="assets/countViews.svg" alt=""><span>213</span></div>
+                <div><img src="assets/countShares.svg" alt=""></div>
+              </div>
+            </div>
+          </div>
+
+          <div class="tweet">
+            <div class="profileImg"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Lesbian_pride_flag_2018.svg/1200px-Lesbian_pride_flag_2018.svg.png" alt=""></div>
+            <div class="content">
+              <span class="profileName">The2ndBliss</span>
+              <span class="handle">2ndBliss</span>
+              <span class="timestamp">3h ago</span>
+              <div class="text">Starting Good Omens S02: 🥰😚😊 vs finishing it: 💀💀💀</div>
+              <div class="postOptions">
+                <div><img src="assets/countReplies.svg" alt=""><span>3</span></div>
```

Comment:
Nice use of span in these buttons. I couldn't make it work. I will try this is the future.

Reply:
That was really more  of a last resort – i TOTALLY forgot the "reblog" etc options and just slapped them in yesterday as an afterthought and just could'nt be bothered, so don't give me too much credit for this haha